github-actions/scan/action.yml

```diff
@@ -105,7 +105,7 @@ runs:
           core.exportVariable('ACTIONS_CACHE_URL', process.env['ACTIONS_CACHE_URL'])
 
     - name: Run SecHub Scan
-      run: node "$GITHUB_ACTION_PATH"/dist/index.js
+      run: exec node "$GITHUB_ACTION_PATH"/dist/index.js
```

Comment:
Maybe a short comment would be nice (above run) - to explain why exec was necessary.